### PR TITLE
[IMP]10720 add module stock_pmc_report.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by elicoidal** [comment]
Headers/README/PEP8

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991

File: stock_pmc_report/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Stock PMC Report',
+    'version': '1.0',
```
Comment (elicoidal) [inline stock_pmc_report/__openerp__.py]: version

File: stock_pmc_report/stock.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+# from openerp.tools.translate import _
```
Comment (elicoidal) [inline stock_pmc_report/stock.py]: remove comments

File: stock_pmc_report/stock.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+# from openerp.tools.safe_eval import safe_eval as eval
+# import openerp.addons.decimal_precision as dp
+# from openerp.tools.float_utils import float_round
+import datetime
+
+
+class stock_count(models.Model):
+    _name = 'stock.count'
+
+    _description = 'statistics stock inventory,On Hand ,Draft,Commit,\
```
Comment (elicoidal) [inline stock_pmc_report/stock.py]: The English rule is comma+space (", ")

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991

**Comment by elicoidal** [comment]
LGTM no test

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**